torch_xla/distributed/xla_dist.py

```diff
@@ -32,6 +33,15 @@ def concat_cmd_list(cmd_list, delimiter=' ', quote='"'):
   return concat
 
 
+def get_metadata(key):
```

Comment:
Also, looks like this is from JAX's TPUVM setup script. To dedupe we can use `ClusterResolver.get_instance_metadata('accelerator-type')`